tests/director/tests/runtime_contexts_api_test.go

```diff
@@ -145,3 +158,127 @@ func TestDeleteRuntimeContext(t *testing.T) {
+
+		subJobStatusPath := resp.Header.Get(subscription.LocationHeader)
+		require.NotEmpty(t, subJobStatusPath)
+		subJobStatusURL := conf.SubscriptionConfig.URL + subJobStatusPath
+		require.Eventually(t, func() bool {
+			return subscription.GetSubscriptionJobStatus(t, httpClient, subJobStatusURL, subscriptionToken) == subscription.JobSucceededStatus
+		}, subscription.EventuallyTimeout, subscription.EventuallyTick)
+		t.Logf("Successfully created subscription between consumer with subaccount id: %q and tenant id: %q, and provider with name: %q, id: %q and subaccount id: %q", subscriptionConsumerSubaccountID, subscriptionConsumerTenantID, providerRuntime.Name, providerRuntime.ID, subscriptionProviderSubaccountID)
+
+		t.Log("Assert provider runtime is visible in the consumer's subaccount after successful subscription")
+		consumerSubaccountRuntimes := fixtures.ListRuntimes(t, ctx, certSecuredGraphQLClient, subscriptionConsumerSubaccountID)
+		require.Len(t, consumerSubaccountRuntimes.Data, 1)
+		require.Equal(t, consumerSubaccountRuntimes.Data[0].ID, providerRuntime.ID)
+
+		t.Log("Assert subscription provider application name label of the provider runtime exists and it is the correct one")
+		subProviderAppNameValue, ok := consumerSubaccountRuntimes.Data[0].Labels[conf.RuntimeTypeLabelKey].(string)
+		require.True(t, ok)
+		require.Equal(t, conf.SubscriptionProviderAppNameValue, subProviderAppNameValue)
+
+		t.Log("Assert there is a runtime context(subscription) as part of the provider runtime")
+		require.Len(t, consumerSubaccountRuntimes.Data[0].RuntimeContexts.Data, 1)
+		require.Equal(t, conf.SubscriptionLabelKey, consumerSubaccountRuntimes.Data[0].RuntimeContexts.Data[0].Key)
+		require.Equal(t, subscriptionConsumerTenantID, consumerSubaccountRuntimes.Data[0].RuntimeContexts.Data[0].Value)
+
+		t.Log("Assert the runtime context has label containing consumer subaccount ID")
+		consumerSubaccountFromRtmCtxLabel, ok := consumerSubaccountRuntimes.Data[0].RuntimeContexts.Data[0].Labels[conf.ConsumerSubaccountLabelKey].(string)
+		require.True(t, ok)
+		require.Equal(t, subscriptionConsumerSubaccountID, consumerSubaccountFromRtmCtxLabel)
+
+		t.Log("Assert provider runtime is visible in the consumer's account after successful subscription")
+		consumerAccountRuntimes := fixtures.ListRuntimes(t, ctx, certSecuredGraphQLClient, subscriptionConsumerAccountID)
+		require.Len(t, consumerAccountRuntimes.Data, 1)
+		require.Equal(t, consumerAccountRuntimes.Data[0].ID, providerRuntime.ID)
+		require.Len(t, consumerSubaccountRuntimes.Data[0].RuntimeContexts.Data, 1)
+
+		t.Log("Assert the consumer cannot modify(create, update or delete) the provider runtime(owner false check)")
```

Comment:
I think the `create` is not applicable for this scenario/asssert because we validate the consumer cannot "modify" the **_already_** created runtime by the provider. However, the delete scenario could be a valid one - the consumer should not be able delete the provider runtime.